test/models/onnx/provenance_input_tags.prototxt

```diff
@@ -10,7 +10,7 @@ graph {
   }
   name: "test_graph"
   initializer {
-    dims: 0
```

Comment:
Since your code have a support for this case its better to leave it as is and add one extra test & model.

Reply:
I've added a test for Constants produced from initializers which are scalars. I will leave this change in this particular provenance tags testing model though - it reflects better how those tags should be constructed now that dynamic shapes support is added.